ds-collector/ds-collector

```diff
@@ -339,7 +339,9 @@ s3_push() {
         -H "Authorization: AWS ${keyId}:${signature}" \
         --retry 9 --retry-connrefused --retry-delay 1 \
         https://"${bucket}.s3.amazonaws.com/${issueId%-*}/${issueId}-${clusterName}-${timestamp}/${dstFileName}" ; do
-    echo "curl … ${srcFilePath} … failed… trying again… "
+    ((c++)) && ((c==10)) && c=0 && break
```

Comment:
Only feedback is that we have 2 levels of retry, the curl retry which handles some transient errors, and then then the overall command retry.

Reply:
Yup. Having learnt from the ci-cassandra.a.o (cassandra-builds) scripts, this is required/optional.